t.css

```
@@ -0,0 +1,5 @@
+a {}
+@media {}
+
+b {}
+/* comment */
```

Expected empty line before comment.

```suggestion

/* comment */
```